minecraft-server/minecraftd.sh

```diff
@@ -180,20 +183,20 @@ server_start() {
 		fi
 
 		# Start the idle server daemon
-		if ${SUDO_CMD} screen -S "${IDLE_SESSION_NAME}" -Q select . > /dev/null; then
-			${SUDO_CMD} screen -S "${IDLE_SESSION_NAME}" -X quit
+		if socket_has_session; then
+			${SUDO_CMD} tmux -L "${IDLE_SESSION_NAME}" send-keys -t0:0.0 "quit" Enter
```

Comment:
I just noticed this now but I think `send-keys` actually does something different than GNU screen's `-X`. The former, I think, sends the keys directly to the process running the session:pane.window while the latter evaluates the command as terminal multiplexer command. Here we want to kill the session and not just print `quit` to the process running within.
Feel free to correct me if I am wrong though!

Reply:
Yeah, that's correct.
I didn't realize that was how you sent commands to screen, I just assumed that was an interaction with the idle server daemon.
I'll replace that with the proper `tmux kill-session` command.